pkg/minikube/machine/filesync_test.go

```diff
@@ -22,11 +22,30 @@ import (
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
+	"k8s.io/minikube/pkg/minikube/assets"
 	"k8s.io/minikube/pkg/minikube/localpath"
 	testutil "k8s.io/minikube/pkg/minikube/tests"
 	"k8s.io/minikube/pkg/minikube/vmpath"
 )
 
+func collectAssets(t *testing.T, root, dest string, flatten bool) []assets.CopyableFile {
```

Comment:
@bobsira add comment, why do we need this helper in this test?

Reply:
@bobsira this func is replacing "assetsFromDir" in minikube code, shouldnt we just fix the minikube code to do the right thing on windows?